base/src/actions.rs

```diff
@@ -184,6 +189,64 @@ impl Model {
                 delta: -column_count,
             }),
         );
+        let worksheet = &mut self.workbook.worksheet_mut(sheet)?;
+
+        // deletes all the column styles
```

Comment:
@nhatcher Do we still need to worry about garbage collecting styles or does this take care of it when we move cells?